### ISSUE TITLE
Better integration of serialized inventory

### ISSUE THREAD

**Issue by mathchouinard** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1191]
While serializing inventory is a nice feature to have, the part that annoys me a little is when I choose the item during the ticket process. I have to click on the item, choose the serialized item in the list, and then continue. 

An alternative and better process would be to be able to use that "Search part or scan barcode" to directly input the serialized barcode and immediately pass onto the next step. Basically:
1. Click on the "search part or scan barcode" bar
2. Use the barcode scanner and scan the serialized item
3. The barcode scanner already "presses" enter, so ideally, it would automatically go to the next step without further input

Right now, if you input the serialized barcode into the search bar, the item appears, but you must still select the item in the list.

**Comment by Uzair-RepairDesk** [comment]
@usmanalikhan93 
Can you please check the scan option of the barcode for the serialized item on the ticket page to populate the part?

**Comment by usmanalikhan93** [comment]
@Uzair-RepairDesk 
Yes currently we don't have the option to scan serialized item on selecting part while creating a ticket.

**Comment by usmanalikhan93** [comment]
For internal task: 
https://repairdesk.atlassian.net/browse/REP-2646

@Uzair-RepairDesk  FYI
